server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1433,8 +1432,11 @@ public void loadTablet(TInfo tinfo, TCredentials credentials, String lock,
             all.remove(extent);
 
             if (!all.isEmpty()) {
-              log.error("Tablet {} overlaps previously assigned {} {} {}", extent,
-                  unopenedOverlapping, openingOverlapping, onlineOverlapping + " " + all);
+              log.error(
+                  "Tablet {} overlaps a previously assigned tablet. It is possibly due to a recent split. "
+                      + "Possible overlapping tablets:\n" + "Unopened tablet:  {}\n"
+                      + "Opening tablet:   {}\n" + "Online tablet:    {}",
```

Comment:
I think it is generally good practice to keep the log message to one line. Some tools may expect log messages to be on one line. If you did that, you could shorten the message a lot, not having to worry about including extra spaces and when you list the "Possible overlapping tablets" you don't need print "tablet" for each type.